frontend/src/containers/Notification.js

```diff
@@ -5,12 +5,25 @@ import resetNotifications from '../actions/notification/reset';
 import Icon from '../components/Icon';
 
 class Notification extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {closed: false};
```

Comment:
If notifications state is already handled by the Redux store, I didn't understand why a React state is also needed